code-of-conduct.html

```diff
@@ -1,7 +1,7 @@
 ---
 layout: base
 title: CODEVID-19 - Code of Conduct
-description: CODEVID-19s Code of Conduct.
```

Comment:
These changes aren't part of changing the privacy policy that I can see.

Can you extract these into a separate PR as they look ready to merge.